demisto_sdk/commands/validate/tests/IN_validators_test.py

```diff
@@ -5963,3 +5966,458 @@ def test_IsValidFeedExpirationPolicy_fully_fetched_feed_with_suddenDeath():
         IsValidFeedExpirationPolicyValidator().obtain_invalid_content_items([feed])
     )
     assert len(validation_results) == 0
+
+
+## -------------------------  IsNewRequiredParamNoDefaultIntegrationValidator IN164 Tests ------------------------- ##
+@pytest.mark.parametrize(
+    "content_items, old_content_items, expected_number_of_failures, expected_msgs",
+    [
+        # Case 1: Valid - new integration with required params (should pass)
+        (
+            [
+                create_integration_object(
+                    paths=["configuration"],
+                    values=[[{"name": "param1", "required": True}]],
+                )
+            ],
+            [],  # No old content (new integration)
+            0,
+            [],
+        ),
```

Comment:
this case is irelevant as the valdation filters out added content items.

Reply:
right, was thinking about covering in case someone changes in future. no need?

Reply:
Nope, we're good.

Reply:
test removed